skyvern/forge/sdk/api/llm/config_registry.py

```diff
@@ -72,3 +72,11 @@ def get_config(cls, llm_key: str) -> LLMRouterConfig | LLMConfig:
             True,
         ),
     )
+if SettingsManager.get_settings().ENABLE_GEMINI:
+    LLMConfigRegistry.register_config(
+        "GEMINI_GPT4V", 
+        litellm.completion("gpt-4-vision-preview",
```

Comment:
should this be a `LLMConfig`?
tagging @ykeremy for review

Reply:
@ykeremy  Also, it looks like pre-commit CI failed to catch all the code style / linter issue from line 78 to line 82

Reply:
is there any specific code formatter is using this project

Reply:
@unaisshemim I'm talking about linters like black and autoflake in the [precommit file](https://github.com/Skyvern-AI/skyvern/blob/main/.pre-commit-config.yaml). 

I see that we only run those precommit after PR is reviewed: https://github.com/Skyvern-AI/skyvern/actions/runs/8534490236

Reply:
> is there any specific code formatter is using this project and they ask me to do with litellm instead of LLMConfig

no linter in this project that enforce how to use litellm.  for [LLMConfigRegistry.register_config](https://github.com/Skyvern-AI/skyvern-cloud/blob/3ffc912b6dc9e7db5c85e68b2380f6549668e67b/skyvern/forge/sdk/api/llm/config_registry.py#L29), the second input should be "LLMRouterConfig | LLMConfig"